src/components/activities/searchPatientActivity/SearchPatientActivity.tsx

```diff
@@ -144,7 +151,7 @@ const SearchPatientActivity: FunctionComponent<TProps> = ({
                 </div>
               </div>
             </div>
-          </div>
+          </form>
           <div className="searchPatient__results">
             <div className="searchPatient__results_count">
               Results: <strong>3</strong>
```

Comment:
We can use the array in `searchPatientResults` to generate the list of patients by implementing the component `PatientSearchItem`. For doing so, we apply the array method `map` on `searchPatientResults` inside the JSX.